app/src/main/java/com/keylesspalace/tusky/ListsActivity.kt

```diff
@@ -143,6 +143,18 @@ class ListsActivity : BaseActivity(), Injectable, HasAndroidInjector {
         editText.text?.let { editText.setSelection(it.length) }
     }
 
+    private fun showListDeleteDialog(list: MastoList) {
+        AlertDialog.Builder(this)
+                .setMessage(getString(R.string.dialog_delete_list_warning, list.title))
+                .setPositiveButton(R.string.action_delete){ _, _ ->
+                    viewModel.deleteList(list.id)
+                }
+                .setNegativeButton(android.R.string.cancel) { d, _ ->
```

Comment:
`.setNegativeButton(android.R.string.cancel, null)`

(change it for the other dialog as well, please)